ntbs-service/Models/NtbsContext.cs

```diff
@@ -51,7 +51,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             if (!optionsBuilder.IsConfigured)
             {
-                optionsBuilder.UseLazyLoadingProxies().UseSqlServer("name=ntbsContext");
```

Comment:
As discussed on slack at some point we're not actually using lazy loading so no need to have this